subprojects/model-core/src/main/java/org/gradle/internal/metaobject/BeanDynamicObject.java

```diff
@@ -415,6 +418,19 @@ public DynamicInvokeResult setProperty(final String name, Object value) {
             return setOpaqueProperty(metaClass, name, value);
         }
 
+        private void trySetReadOnlyProperty(String name, Object value, MetaBeanProperty metaBeanProperty) {
```

Comment:
🤔 We could improve this name. It's not that the property is read-only, but that it's getter-only, no? I think the name should reflect that.